lib/tasks/source_map.rake

```diff
@@ -104,7 +104,7 @@ end
 task :reload_bea_series_only => :environment do
   t = Time.now
   #could also hard code this...
-  bea_series = Series.get_all_series_from_website('bea.gov')
+  bea_series = Series.get_all_series_from_website('load_from_bea')
```

Comment:
we do want the old bea.gov series since there are still a lot of source that either haven't yet or can't be migrated to the API.